strax/plugins/plugin.py

```diff
@@ -110,6 +110,8 @@ def __init__(self):
             raise ValueError(f"depends_on not provided for {self.__class__.__name__}")
 
         self.depends_on = strax.to_str_tuple(self.depends_on)
+        # Remove duplicates
+        self.depends_on = tuple(dict.fromkeys(self.depends_on))
```

Comment:
Better use `tuple(set(self.depends_on))`.

Reply:
I believe it's the same functionally, but yes, let's make it easier.